tools/diann/diann.xml

```diff
@@ -0,0 +1,486 @@
+<tool id="diann" name="DIA-NN" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@">
+    <description>is a software for DIA/SWATH data processing</description>
+    <macros>
+        <token name="@TOOL_VERSION@">1.8.1</token>
+        <token name="@VERSION_SUFFIX@">0</token>
+    </macros>
+    <requirements>
+        <container type="docker">biocontainers/diann:v@TOOL_VERSION@_cv1</container>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        mkdir ./input_data &&
+        mkdir ./tmp &&
+
+        ## generate string like "--f file1 --f file2 ..." for each input in $input.f, which are comma separated
+        #set infiles_str = ''
+        #for $infile in $input.f
+            ## if brukertdf.d.tar, extract to ./input_data and append "./input_data/" to infiles_str
+            #if $infile.is_of_type("brukertdf.d.tar"):
+                tar -xf '$infile' -C ./input_data &&
+                #set $infiles_str += '--f ./input_data/' + $infile.element_identifier [:-4] + ' '
+            #else
+                #set $infiles_str += '--f ' + $infile.element_identifier  + ' '
```

Comment:
So this will end up as `--f small-peakpicking-cwt-allMS`. But no path is ever passed to the program, or did I miss that? How should the tool know where the file is located?

Reply:
You are right! fixed